tests/server/cluster/cluster_test.go

```diff
@@ -186,7 +186,7 @@ func TestRegionStatistics(t *testing.T) {
 	re := require.New(t)
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
-	tc, err := tests.NewTestCluster(ctx, 2)
+	tc, err := tests.NewTestCluster(ctx, 3)
```

Comment:
Why add a new server?

Reply:
Because this test has only 2 pd's and five ResignLeader times, whenever there is a network or disk problem will cause the lease to expire, and result in one more leader resigning.
And then there is a high probability of being recognized as frequent campaign times to transfer leader failed